plugins/tiddlywiki/highlight-legacy/howto.tid

```diff
@@ -0,0 +1,10 @@
+title: $:/plugins/tiddlywiki/highlight/howto
```

Comment:
I think either we should include this information in the existing "readme" or "usage" tiddlers. If it's better in it's own tab, perhaps it might be better called "languages"?

Reply:
moved to "usage"